package.json

```diff
@@ -92,10 +92,8 @@
         "tabular-json": "0.1.3",
         "tldts": "^5.6.2",
         "underscore": "1.8.3",
-        "useragent": "^2.3.0",
         "web-ext": "^2.7.0",
-        "xvfb": "0.4.0",
-        "yamlparser": "0.0.2"
```

Comment:
I don't see this being imported anywhere in the removed code - this was a dead dependency?

Reply:
useragent uses it internally and the Grunt build process complained if it wasn't included. (kinda the issue at hand here, we got require and yamlparser included into useragent when we were not ever wanting them)